### ISSUE TITLE
Play (short) audios simultaneously

### ISSUE THREAD

**Issue by burak-yildirim** [florent37/Flutter-AssetsAudioPlayer#19]
**TL;DR** I want to play different short audios without blocking each other if any of them is played before the last one finishes. Down below i explained what i did in more detail.

---

I have different play buttons on the same screen and each have different ```AssetAudioPlayer``` instance. Each button has different instance because i load the audios(```_assetAudioPlayer.open(_assetAudio)```) on my 'Loading' page to prevent delays while playing. Unfortunately, all the buttons play the same audio which loaded last. Also i can't play again before the last audio finishes.

I realized that ```_assetAudioPlayer.open(_assetAudio)``` happens very quickly so i can repeat it at every play event. But this time it stops other play events if there is any. 

Is it possible to achieve playing non-blocking audios without trading off performance?

**Comment by florent37** [comment]
fixed in https://github.com/florent37/Flutter-AssetsAudioPlayer/pull/39
